homeassistant/components/media_extractor/__init__.py

```diff
@@ -51,7 +53,12 @@ def play_media(call: ServiceCall) -> None:
         DOMAIN,
         SERVICE_PLAY_MEDIA,
         play_media,
-        schema=cv.make_entity_service_schema(MEDIA_PLAYER_PLAY_MEDIA_SCHEMA),
+        schema=cv.make_entity_service_schema(
+            {
+                **MEDIA_PLAYER_PLAY_MEDIA_SCHEMA,
+                vol.Optional(ATTR_COOKIES_FILEPATH, default=""): cv.string,
```

Comment:
Should this be part of each individual service call? Feels more like a general configuration option to me?

Reply:
It should be part of each service call. Cookies are typically exported per-domain (e.g. twitch.tv). I wouldn't expect users to have cookies for all of the sites they stream from in one file.